README.md

```diff
@@ -18,45 +16,168 @@ Make a pull request to add to this collaborative tale about Harry and his friend
 
-5. ###### Add your changed file to the staging area
+
+6. ###### Add your changed file to the staging area
 
 
   ```sh
   $ git add fan-fiction.md
   ```
 
-6. ###### Commit your changes with a sweet commit message
+  This puts fan-fiction.md into the **staging area**. If you make another change, it will be removed from it.  
+
+  To see what you've changed and what is in your **staging area**, you can type `git status`.
+
+
+7. ###### Commit your changes with a sweet commit message
 
 
   ```sh
   $ git commit -m 'Hermione discovers who sent the valentines day card'
   ```
 
+  A commit is the point at which the state of everything in the **staging area** is recorded. 
 
-7. ###### Push your changes to a branch on github
+8. ###### Push your changes to a branch on github
 
 
   ```sh
-  $ git push origin <your_branch_name>
+  $ git push origin your_branch_name
   ```
 
+  This copies your branch information to the repository on GitHub.
+
+  You should be able to see your branch [here](https://github.com/railsgirlswgtn/fan-fiction-github-challenge/branches).
+
 
-8. ###### Create a pull request to master on github
+9. ###### Create a pull request to master on github
 
 
   ![](http://i.imgur.com/zZPQLw0.png)
+
+  Go through the steps to create a pull request on GitHub. Genevieve, Rob, and Merrin will merge pull requests into master.
+
+#### Updating your repository
+  Hopefully this repository will be full of delightful contributions to literature. That means that you will have to **pull** from **origin** to get everyone elses' commits.
+
+  1. First of all, if you are on a branch, make sure you have **committed** all of the changes you want to make. When you type `git status`, you want it to say: 'nothing to commit, working directory clean'. 
+    * There are more detailed instructions above about `git add` and `git commit`.
+    * If you don't want to commit but you do want to switch branches, you can use `git stash` to hide the changes you have made and `git stash pop` to bring them back.
+
+  2. Next, check out the master branch: `git checkout master`. 
+
+  3. Your task now is to `git pull`. 
+    * As long as your working directory is clean and you have only made commits to your branch and not to master, this should just pull the changes down from GitHub that everyone else has made. Your master branch should now look like the master branch on GitHub.
+    * If you have made commits to master, you might get merge conflicts here. Skip ahead to point 8 for tips on resolving merge conflicts, and find a coach to get some help with getting your commits onto a branch. 
+
+  4. Now head back to the branch that your commits are on: `git checkout whatever_your_branch_is_called`.
+
+  5. The next part is hard. Are you ready for it?
+
+  6. Are you sure? 
+
+  7. Ok. Take a deep breath and type `git merge master`.
```

Comment:
If people are cloning the repository locally they will need to pull down to update. 

Maybe this section could be more explicitly flagged as optional but otherwise we'll have a lot of pull requests that need a lot of work before they're ready

The merge instructions may be a little over the top because I am extremely frightened of merge conflicts and writing piece by piece instructions soothes my fear